deepchem/models/torch_models/layers.py

```diff
@@ -4794,6 +4798,46 @@ def forward(self, dist: torch.Tensor) -> torch.Tensor:
         return output
 
 
+class VariationalRandomizer(nn.Module):
```

Comment:
Make sure to add to layers.rst and torch_layers.csv

Reply:
Done